de.tubs.variantsync.core-test/test/de/tubs/variantsync/patch/base/DiffUtilsTest.java

```diff
@@ -54,6 +54,7 @@ public void setUp() throws Exception {
 	public void testPatch1() throws DiffException {
 		Patch<String> patch = DiffUtils.diff(lines1, lines3, 1);
 		assertTrue(patch.getDeltas().size() == 3);
+		

```

Comment:
Why did you add the empty line here? Please fix.

Reply:
fixed: https://github.com/tthuem/VariantSync/pull/53/commits/02b4399e94925cbea7ba19cd021e9e846522bc85